.github/workflows/weekly-preview.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         export YEAR_WEEK=$(date +'%y%U')
         echo "Year week for tag is ${YEAR_WEEK}"
         if ! [[ $YEAR_WEEK =~ ^[0-9]{4}$ ]] ; then echo "Wrong 'year week' format.  Should be 4 digits."; exit 1 ; fi
-        git tag "0.9.dev${YEAR_WEEK}"
+        git tag "0.10.dev${YEAR_WEEK}"
```

Comment:
Should we use `0.10` or `0.9.1` or `1.0` for the next?
I am not very sure.

Thanks.